### DISCUSSION TITLE
求 starter 的 blazor 版本

### DISCUSSION THREAD

**Discussion by fixnil** [AchievedOwner/TDesignBlazor#87] [讨论]
作者牛逼！！！

第一眼看到 TDesign 就感觉这框架可处，最近在做一个 Blazor 项目特别想用，没想到这里就有了；
希望赶紧有一个 Blazor 版本的 Starter，或者作者可以提交一个项目框架，说明怎么做，我们去填充内容。

文档地址：https://tdesign.tencent.com/vue-next/quick-start  
预览地址：https://tdesign.tencent.com/starter/vue-next

**Comment by teacher-zhou** [comment]
感谢支持，关注[里程碑](https://github.com/AchievedOwner/TDesignBlazor/milestones)可获得最新的进度
